run_uh.py

```diff
@@ -121,22 +121,18 @@ def excepthook(exception_type, value, tb):
 		with open(outfilename, 'a') as f:
 			traceback.print_exception(exception_type, value, tb, file=f)
 		traceback.print_exception(exception_type, value, tb)
-		print('')
-		print(T('Unknown Horizons has crashed.'))
-		print('')
+		print('\n', T('Unknown Horizons has crashed.', '\n'))
```

Comment:
I would always favor single print() calls, though I agree that here it doesn't matter so much. You can kick this commit out if you think it's redundant :).